.github/workflows/nuget-audit.yml

```diff
@@ -0,0 +1,19 @@
+name: NuGet Audit
+on:
+  workflow_dispatch:
+env:
+  DOTNET_NOLOGO: true
+jobs:
+  build:
+    name: Linux
+    runs-on: ubuntu-22.04    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.2.2
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v4.1.0
```

Comment:
I'm not so sure it was a good idea to use full versions here. This will result in this repo and all synced repos to get Dependabot PRs for this. It's not a CI so we don't care about repeatable builds, just that it runs. It would be more maintainable for both of these to be simply `@v4`.

Reply:
This isn't the final version. I will fix it when we deploy the next version.